backend/src/main/resources/application.yml

```diff
@@ -15,3 +15,6 @@ githubFeignClient:
 
 jira:
   url: https://demo.atlassian.net
+
+buildKite:
+  url: 54.223.108.141:4323
```

Comment:
Should not pass fixed IP into application.yaml
PLZ ref to https://stackoverflow.com/questions/23027315/does-application-yml-support-environment-variables to change